### ISSUE TITLE
Few changes to copying private URL-s

### ISSUE THREAD

**Issue by lbownik** [CeON/fairchive#2683]
The problems occur on dataset page:

![Image](https://github.com/user-attachments/assets/dde26c21-c5a9-4db2-9fe2-2b2a843b7273)

And on modal window for URL management:

![Image](https://github.com/user-attachments/assets/b3b56473-eb07-47b2-a4d2-f2872211bceb)

        
mainly for WCAG reasons - we should prefer text over graphic representation, so it would be reasonable to add a text "Copy URL to clipboard" by the icon.

maybe we could change the icon to the more recognized "Copy" icon, namely something like:

![Image](https://github.com/user-attachments/assets/ad479589-55d6-4205-9a05-f50f6bcd894b)

![Image](https://github.com/user-attachments/assets/668dc3c1-f856-44e1-ae5b-3a573f23d830)

(I'm not sure if it is available among our icon stack).

      
Currently it is impossible to tab into the icon. It should be.
Are there appropirate ARIA labels? If not, there should be.
